src/components/ButtonSelectGroup/ButtonSelectGroup.js

```diff
@@ -66,6 +67,15 @@ export const ButtonSelectGroup = ({
   // Set up validation hooks
   const [getError, setError, , validate] = useErrorMessage(validator)
 
+  // can set current value if selectedValue is not selected.  This is important for gatsby rendering
+  if (
+    !isUndefined(currentValue) &&
```

Comment:
@seanhho what does `isUndefined` check exactly? is it redundant that we are checking both `!isUndefined(currentValue)` and `currentValue !== ''`?

Reply:
Looks like `isUndefined` checks for null or undefined of the value, so my understanding is that `!currentValue` covers all three scenarios: null, undefined, and empty string. Is there anything that I miss here?

Reply:
yes this is true.  Notably in the form.js component it passes in an empty string where as in most situations we pass in nothing/undefined.  I make sure to use the lodash function because javascript has weird issues with undefined that we want to avoid.